styles.css

```diff
@@ -26,4 +26,12 @@ li a {
   padding: 20px 16px;
   text-decoration: none;
   font-family: "Courier New", Courier, monospace
+}
+
+section {
+  max-height: 800px;
+  max-width: 800px;
+  text-align: justify;
```

Comment:
gotcha. This started with a "why..?" that I didn't know the answer to. I understand now there is another way to make this look better. I guess my future commit can be centering and aligning my section text. coming up!